protocol/0029-FEES-fees.md

```diff
@@ -82,6 +82,8 @@ Note, discounts are calculated and applied one after the other and **before** re
 - infrastructure: staking/governance system/engine (network wide)
 - maker: market framework / market making (network wide)
 - liquidity: market making system (per market)
+- treasury: Fees sent to network treasury for later usage via governance votes (network wide)
+- buyback: Fees used to purchase governance tokens to the protocol via regular auctions (network wide)
```

Comment:
do we need a new account type for holding those fees?

Reply:
Resolving as covered in treasury / buyback fees PR.